ee/desktop/user/universallink/handler_other.go

```diff
@@ -0,0 +1,39 @@
+//go:build !darwin
+// +build !darwin
+
+package universallink
+
+import (
+	"log/slog"
+)
+
+// On other OSes, universal link handling is a no-op.
+type noopUniversalLinkHandler struct {
+	unusedInput chan string
```

Comment:
That's the method signature. Why is it in the struct?

Reply:
Since `universallink.NewUniversalLinkHandler(slogger)` needs to return a channel, it felt tidier to keep it in the struct and close it on interrupt like the non-noop handler does.